### PR TITLE
ManipulationCompleted might not be raised for each ManipulationStarted

### PR DESCRIPTION
**Pull request by dr1rrb**
GitHub Issue: #5929 

## Bugfix
`ManipulationCompleted` might not be raised for each `ManipulationStarted`.

## What is the current behavior?
When a `ScrollViewer` kicks-in (and stole the pointer) the `PointerCancelled` event is not propagated to elements which are not under the pointer, preventing the `GestureRecognizer` to complete.

## What is the new behavior?
Pointer cancelled events are sent to `GestureRecognizer` in any cases.

## PR Checklist
- [ ] ~~Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)~~
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Manipulation_Tests.cs [hunk 1]
```diff
@@ -111,11 +111,30 @@ public void Manipulation_WithNestedElement()
 			Assert.IsTrue(result.Contains("[PARENT] Manip delta[CHILD] Pointer moved[PARENT] Manip delta[CHILD] Pointer moved"));
 		}
 
-		private static (Point position, ManipulationDelta delta, ManipulationDelta cumulative) Parse(string raw)
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)]
+		[Ignore("We need to do a L manipulation to get manip started then pt cancel due to SV kick-in")]
```
Comment (davidjohnoliver) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Manipulation_Tests.cs:117]: psshh yeah obviously

**Review by davidjohnoliver — APPROVED** [review]

**Review by carldebilly — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.7

**Pull request merged**